kolibri/auth/api.py

```diff
@@ -135,18 +135,23 @@ class LearnerGroupViewSet(viewsets.ModelViewSet):
 
 class SignUpViewSet(viewsets.ViewSet):
 
+    def extract_request_data(self, request):
+        return {
+            "username": request.data.get('username', ''),
+            "full_name": request.data.get('full_name', ''),
+            "password": request.data.get('password', ''),
+            "facility": Facility.get_default_facility().id,
+        }
```

Comment:
This logic was extracted out from the `create` method so that it could be overridden cleanly in the plugin, without having to redefine the entire `create` method.